src/cache/listenedChannels.ts

```diff
@@ -0,0 +1 @@
+export const listenedChannels: Set<string> = new Set()
```

Comment:
I prefer use eslint/prettier for such cases.

You can contribute, if you wan't to.

Reply:
ESLint lints JavaScript, not general whitespaces.

Maybe later.

Reply:
There is rule for that. https://eslint.org/docs/rules/eol-last

And you are right, it will lint only ts/js.

You can also add editorconfig, if you wan't to.

Reply:
> There is rule for that. https://eslint.org/docs/rules/eol-last

I'm surprised.